bogenliga/src/app/modules/shared/services/current-user/current-user.service.ts

```diff
@@ -3,17 +3,16 @@ import {Router} from '@angular/router';
 import {select, Store} from '@ngrx/store';
 import {isNullOrUndefined} from '@shared/functions';
 import {filter, map} from 'rxjs/operators';
-import {LocalDataProviderService} from '../../local-data-provider/services';
+import {LocalDataProviderService} from '@shared/local-data-provider';
 import {AppState, Login, Logout} from '../../redux-store';
 import {UserState} from '../../redux-store/feature/user';
 import {Notification, NotificationUserAction} from '../notification/types';
 import {UserPermission} from './types/user-permission.enum';
 import {UserSignInDTO} from './types/user-sign-in-dto.class';
-import {CredentialsDO} from '@user/types/credentials-do.class';
 
 const CURRENT_USER_KEY = 'current_user';
 const LOGIN_EMAIL_KEY = 'login_email';
-const DEFAULTUSERRNAME = 'ligadefault';
+const DEFAULTUSERNAME = 'ligadefault';
```

Comment:
Kannst du die Konstante noch umbenennen in DEFAULT_USERNAME? Das ist leichter zu lesen. Danke